demo/addons/godot-openxr/scenes/first_person_controller_vr.gd

```diff
@@ -74,13 +93,21 @@ func _get_xr_viewport() -> Viewport:
 	else:
 		return get_viewport()
 
-func _start_passthrough():
-	if start_passthrough:
-		# make sure our viewports background is transparent
-		_get_xr_viewport().transparent_bg = true
+func is_passthrough_supported() -> bool:
+	return enabled_extensions.find("XR_FB_passthrough") >= 0
+
+func _start_passthrough() -> bool:
+	# make sure our viewports background is transparent
+	_get_xr_viewport().transparent_bg = true
```

Comment:
Is there a way to detect the version of Godot being used and print a warning for https://github.com/GodotVR/godot_openxr/issues/139?

Reply:
We should have access to the version number, I'll need to check.

Reply:
Added that in with a few other checks that seemed to make sense.